guides/hydra-box/readme.md

```diff
@@ -6,7 +6,13 @@ Hydra-box is express.js middleware developed in [Zazuko GmbH][zazuko] and led by
 It is still in an early stage so currently do have a look at its repository
 [zazuko/hydra-box](https://github.com/zazuko/hydra-box).
 
-More content will come here as soon as the API solidifies a little.
+## Installation
```

Comment:
I know it's not part of this PR but I think it could be: at the beginning of this file:

```diff
- Hydra-box is express.js middleware developed in [Zazuko GmbH][zazuko] and led by
+ Hydra-box is an express.js middleware developed at [Zazuko GmbH][zazuko] and led by
```
(Official name would be "Express" but without a link to people it could confuse people)